Formula/botan.rb

```diff
@@ -14,7 +14,7 @@ class Botan < Formula
 
   depends_on "pkg-config" => :build
   depends_on :macos # Due to Python 2
```

Comment:
Is this still true? The configure script suggests it should work with python 3.

Reply:
Upstream maintainer here. The configure script absolutely works with Python3 (we are maintaining 2+3 compat for the time being, though will eventually move to Python3 only)

Reply:
@securitykernel could you amend your PR to make it use `python@3.8` then instead of depending on macOS?

Reply:
Did that.

Reply:
You might need to set the python version cause it's installing python 2.7 bindings right now.

Reply:
I see. Updated.

Reply:
@SMillerDev Rebased onto master (2.15.0 version bump). Ready for merge IMHO.